src/common/internal/test_group.ts

```diff
@@ -0,0 +1,414 @@
+import { Fixture, SkipTestCase, TestParams, UnexpectedPassError } from '../framework/fixture.js';
```

Comment:
This file has not changed meaningfully, it just didn't get tracked as a rename because I made a new file with the same name.